backend/utils/createListing.js

```diff
@@ -73,20 +85,29 @@ async function createListing({ network, pk, title }) {
     throw new Error(`Error adding listing to ${network.ipfsApi}`)
   }
 
-  let tx
-  try {
-    tx = await Marketplace.methods
-      .createListing(ipfsBytes, 0, account.address)
-      .send({ from: account.address, gas: 350000 })
-  } catch (e) {
-    log.error(e)
-  }
-
-  web3.currentProvider.connection.close()
+  // Send the transaction to the network and wait for it to get mined.
+  const options = { gasLimit: 350000 }
```

Comment:
Second time I've seen this.  Might be good to have gas constants somewhere.

Reply:
Moved to a common file: packages/src/marketplace.js